docs/content/latest/yugabyte-platform/back-up-restore-universes/back-up-universe-data/ycql.md

```diff
@@ -34,25 +34,17 @@ Use Yugabyte Platform to back up your YugabyteDB universe YCQL data.
 
 To schedule backups for a later time, or as a recurring task, see [Schedule universe YCQL data backups](../../schedule-data-backups/ycql).
 
-To immediately back up your YugabyteDB universe YCQL data:
+To immediately back up your YugabyteDB universe YCQL data, perform the following:
 
-1. Open the **Universe Overview** and then click the **Backups** tab. The **Backups** page appears.
+1. Open your universe and select **Backups**.
 1. Click **Create Backup** to open the **Create Backup** dialog.
 
     <br/><br/>
 
-    ![Create Backup - YCQL](/images/yp/create-backup-ycql.png)
+    ![Create Backup - YCQL](/images/yp/create-backup-ycql.png)<br><br>
 
-1. Click the **YCQL** tab and enter the following:
+1. Complete the fields presented in the **YCQL** tab.
```

Comment:
Shouldn't we extend this part to describe that e.g. storage types list depends on the existing backup storage configurations available ?

Reply:
Added info